src/init/features/dataconnect/index.ts

```diff
@@ -196,7 +197,11 @@ async function writeFiles(config: Config, info: RequiredInfo) {
     for (const f of info.schemaGql) {
       await config.askWriteProjectFile(join(dir, "schema", f.path), f.content);
     }
+  } else {
+    // Even if the schema is empty, lets give them an emty .gql file to get started.
+    fs.ensureFileSync(join(dir, "schema", "schema.gql"));
```

Comment:
Do you think it's worth also filling in that schema with the commented out schema? I could go either way